tests/test_runtime_only/test_literal_type.py

```diff
@@ -1,10 +1,12 @@
-from typing import Literal, Union
+from typing import Union
+
+from typing_extensions import Literal
 
 from basedtyping.runtime_only import LiteralType
 
 
 def test_literal_type_positive() -> None:
-    assert isinstance(Literal[1, 2], LiteralType)
+    assert isinstance(Literal[1, 2], LiteralType)  # type: ignore[no-any-expr]
```

Comment:
why does this have `Any` in it now?

Reply:
I think it's a 3.7 moment

Reply:
in that case wouldn't you need to do this so yiou don't get an error on newer python versions
```suggestion
    assert isinstance(Literal[1, 2], LiteralType)  # type: ignore[no-any-expr,unused-ignore]
```

Reply:
Oh, it's actually because this used to be `typing.Literal` and now it's `typing_extensions.Literal`, which is typed differently.